nemo/src/execution/planning/plan_head_restricted.rs

```diff
@@ -113,6 +113,7 @@ impl HeadStrategy for RestrictedChaseStrategy {
         current_plan: &mut SubtableExecutionPlan,
         variable_translation: &VariableTranslation,
         node_matches: ExecutionNodeRef,
+        _node_aggregates: Option<ExecutionNodeRef>,
```

Comment:
Why is this here? The underscore seems to indicate that this is not used(?)

Reply:
Aggregation unterstützen wir bislang nicht zusammen mit existentiellen Regeln